src/validation_workflow/validation_rpm.py

```diff
@@ -25,3 +29,47 @@ def download_artifacts(self, projects: list, version: str) -> bool:
                     logging.info(f"Invalid URL - {url}")
                     raise Exception(f"Invalid url - {url}")
         return True
+
+    @classmethod
+    def installation(self, args: ValidationArgs) -> bool:
```

Comment:
since the methods `installation`, `start_cluster`, `validation` and `stop_cluster` have to be present for each validation type, wouldn't it be a good idea to make them abstract and make sure they are implemented for all the future distribution types?

Reply:
Another way of saying the same, all `@classmethod` should become instance methods and override common implementations in the parent `Validation` class.

Reply:
So does that mean we need to directly declare classmethods in parent class instead of making them abstract and override them in child classes when required.

Reply:
A class method is a method on the class itself, where the class acts as a namespace. You can call those as `Klass.method`. You almost never need those. An instance method is just a method that can be overridden in a child class. An abstract method just means that all classes in the class hierarchy will have this method, but the implementations are very different, so we don't need to provide a common implementation in the base class.

Reply:
Agree to this. That was the plan. All the future distributions too need to have those methods. Override or pass, depends on the respective distribution.